src/main/kotlin/uk/gov/justice/digital/hmpps/manageusersapi/model/filter/PrisonUserFilter.kt

```diff
@@ -0,0 +1,15 @@
+package uk.gov.justice.digital.hmpps.nomisuserrolesapi.data.filter
```

Comment:
package doesn't match

Reply:
very good point - I'm surprised nothing automatic picked this up